sunpy/net/helio/tests/test_helio.py

```diff
@@ -293,6 +294,14 @@ def test_client_search(client):
     res = client.search(a.Time(start, end), a.helio.TableName(table_name), a.helio.MaxRecords(10))
     assert len(res) == 10
 
+    with pytest.raises(ValueError, match="Helio will only return a max of 20000 results."):
```

Comment:
Can this be made into a seperate offline test? 

It does not need to be added to this unit test.